src/components/radio/radio_group.html

```diff
@@ -0,0 +1 @@
+<ng-content></ng-content>
```

Comment:
Actually, if the template consists of nothing but `<ng-content>`, it should just be a `@Directive` instead of `@Component`.